docs/devel/hello-world-gadget.md

```diff
@@ -311,31 +311,18 @@ description: 'TODO: Fill the gadget description'
 homepageURL: 'TODO: Fill the gadget homepage URL'
 documentationURL: 'TODO: Fill the gadget documentation URL'
 sourceURL: 'TODO: Fill the gadget source code URL'
-tracers:
+datasources:
   open:
-    mapName: events
-    structName: event
-structs:
-  event:
     fields:
-    - name: pid
-      description: 'TODO: Fill field description'
-      attributes:
-        width: 16
-        alignment: left
-        ellipsis: end
-    - name: comm
-      description: 'TODO: Fill field description'
-      attributes:
-        width: 16
-        alignment: left
-        ellipsis: end
-    - name: filename
-      description: 'TODO: Fill field description'
-      attributes:
-        width: 16
-        alignment: left
-        ellipsis: end
+      comm:
```

Comment:
Right, it's coming from the metadata package. I also don't know if we should populate it with all defaults.

Reply:
We either do that or provide a scheme where people can see the default values we are using. Otherwise, users will see columns displayed with a certain format and they'll not understand where such a configuration is coming from.

Reply:
We can document the default annotations we use. The idea is to keep the metadata files clean without duplicating all default values.